conf/modules.config

```diff
@@ -289,7 +289,7 @@ process {
         publishDir = [
             path: { "${params.outdir}/kraken2/${meta.db_name}" },
             mode: params.publish_dir_mode,
-            pattern: '*.{txt}'
+            pattern: '*.{txt,report,fastq.gz}'
```

Comment:
The pattern in the kraken module is:

  ```
   type: file
      description: |
        Kraken2 report containing stats about classified
        and not classifed reads.
      pattern: "*.{report.txt}"
```

Should this be:

`pattern: '*.{txt,fastq.gz}'`

Reply:
txt is the default output, report is the `readclassification` thing (basically gives a list of all reads, and what taxonomic assignment each read was provided), and fastqc.gz is the aligned reads

Reply:
I don' follow the qeustion, sorry - can you rephrase?

Reply:
I was wondering if the parameter `kraken2_save_readclassification` could be named as `kraken2_save_classified_reads` instead?

Reply:
I don't think so, I fear people would think this is actual reads that were classified in e.g. FASTQ format (which is what `save_aligned` is meant to be doing), whereas this is a text file with a list of reads and their taxonomic assingments (and more info about them)

Reply:
Sounds good then.

Reply:
AAwesome, I'll re-request review for you to check you're happy with it now I've added the comments :)